src/main/java/se/kth/sda/skeleton/comments/CommentService.java

```diff
@@ -11,14 +11,19 @@
 import se.kth.sda.skeleton.user.User;
 import se.kth.sda.skeleton.user.UserRepository;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.*;
 import java.time.LocalDateTime;
+import java.util.Collections;
 import java.util.List;
 
 /**
  * Represents the service layer. It contains defined functionalities according to business logic for Comments.
  */
 @Service
 public class CommentService {
+    private static String videoDirectory = System.getProperty("user.dir") + "/frontend/src/videos/comment";
```

Comment:
Should we add the contents of this folder to .gitignore but keep a README so the folder uploads?

Reply:
I think when FE gets implemented we can do that change as it was stuff with react that got bad when fetching those videos